deploy/nginx-ingress/values.yaml

```
@@ -2,3 +2,4 @@
 controller:
   proxySetHeaders:
     X-Original-Host: "$host"
+  progressDeadlineSeconds: null # Workaround: Please see https://github.com/helm/helm/issues/30878
```

Let's add a TODO to look back at this when a patch is out

```suggestion
  progressDeadlineSeconds: null # TODO: Workaround for Helm 3.18.0 regression, remove when fixed, see https://github.com/helm/helm/issues/30878
```